bad.php

```diff
@@ -0,0 +1,6 @@
+<?php
+class SomeClass {
+    public function bad_braces (  ) {
```

Comment:
Tabs must be used to indent lines; spaces are not allowed
Space between function name and opening parenthesis is prohibited.
Missing function doc comment
Expected 0 spaces between brackets of function declaration; 2 found